main.js

```diff
@@ -88,28 +68,55 @@ class Timer {
 }
 
 class DOM_Manager {
-    constructor() {
-        this.cardsCountVar = document.getElementsByClassName("card").length;
+    constructor(Fn) {
+        this.callBackFn=Fn;

```

Comment:
It's not strait-forward for which functionality `callBackFn` is responsible. Better name is needed.
The same for `Fn`